### PR TITLE
Added accion_name and service_ns parameters

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
fixes https://github.com/ipa320/cob_robots/issues/242

File: cob_default_robot_config/cob3-3/sdh_joint_configurations.yaml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 joint_names: ["sdh_knuckle_joint", "sdh_thumb_2_joint", "sdh_thumb_3_joint", "sdh_finger_12_joint", "sdh_finger_13_joint", "sdh_finger_22_joint", "sdh_finger_23_joint"]
+action_name: /gripper_controller/follow_joint_trajectory
```
Comment (floweisshardt) [inline cob_default_robot_config/cob3-3/sdh_joint_configurations.yaml:2]: isn't this called "sdh_controller"?

File: cob_default_robot_config/cob3-6/sdh_joint_configurations.yaml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 joint_names: ["sdh_knuckle_joint", "sdh_thumb_2_joint", "sdh_thumb_3_joint", "sdh_finger_12_joint", "sdh_finger_13_joint", "sdh_finger_22_joint", "sdh_finger_23_joint"]
+action_name: /gripper_controller/follow_joint_trajectory
```
Comment (floweisshardt) [inline cob_default_robot_config/cob3-6/sdh_joint_configurations.yaml:2]: isn't this called "sdh_controller"?

File: cob_default_robot_config/cob3-3/sdh_joint_configurations.yaml (hunk 1 above)
Comment (NadiaHG) [inline cob_default_robot_config/cob3-3/sdh_joint_configurations.yaml:2]: No, it is called gripper_controller: https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_controller_configuration_gazebo/controller/gripper/gripper_controller_sdh.yaml

File: cob_default_robot_config/cob3-6/sdh_joint_configurations.yaml (hunk 2 above)
Comment (NadiaHG) [inline cob_default_robot_config/cob3-6/sdh_joint_configurations.yaml:2]: No, it is called gripper_controller: https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_controller_configuration_gazebo/controller/gripper/gripper_controller_sdh.yaml

File: cob_default_robot_config/cob3-3/sdh_joint_configurations.yaml (hunk 1 above)
Comment (floweisshardt) [inline cob_default_robot_config/cob3-3/sdh_joint_configurations.yaml:2]: ok, this seems to be new then and is an API change. But that's OK for me.

File: cob_default_robot_config/cob3-6/sdh_joint_configurations.yaml (hunk 2 above)
Comment (floweisshardt) [inline cob_default_robot_config/cob3-6/sdh_joint_configurations.yaml:2]: ok, this seems to be new then and is an API change. But that's OK for me.

**Comment by floweisshardt** [comment]
visual review: OK

**Pull request merged**